### PR TITLE
update packages with minor upgrades

### PR DESCRIPTION
**Pull request by emord**
Just the packages that we explictly install that have minor upgrades available. 

Still need go through and verify they follow semantic versioning for each one

@dannyroberts

### CODE REVIEW

**Review by dannyroberts — APPROVED** [review]
Seems in theory a good practice to stay on the latest patch release, thanks for taking the effort!

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,23 +1,23 @@
 # needed to build gevent from source
 cython==0.23.4
 restkit==4.2.2
-iso8601==0.1.10
+iso8601==0.1.11
```
Comment (emord) [inline requirements/requirements.txt]: https://pypi.python.org/pypi/iso8601

File: requirements/requirements.txt [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 jsonobject==0.7.1
 jsonobject-couchdbkit==0.8.0.2
 celery==3.1.18
 # required by django-digest
-decorator==4.0.4
+decorator==4.0.11
```
Comment (emord) [inline requirements/requirements.txt]: https://pypi.python.org/pypi/iso8601

File: requirements/requirements.txt [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 django==1.10.7
 django-braces==1.8.1
 django-celery==3.2.1
-django-crispy-forms==1.6.0
+django-crispy-forms==1.6.1
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/django-crispy-forms/django-crispy-forms/blob/dev/CHANGELOG.md

File: requirements/requirements.txt [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 jsonfield==1.0.3
 dropbox==7.1.1
 defusedxml==0.4.1
 diff-match-patch==20120106
 django-tastypie==0.13.3
 djangorestframework==3.5.4
-djtables==0.1.1
+djtables==0.1.2
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/adammck/djtables/compare/v0.1.1...v0.1.2

File: requirements/requirements.txt [hunk 5]
```diff
@@ -36,19 +36,19 @@ pycrypto==2.6.1
 py-KISSmetrics==1.0.1
 https://github.com/dimagi/files/raw/master/python-requirements/pygooglechart-0.3.0.tar.gz
 python-dateutil==1.5
-python-magic==0.4.10
+python-magic==0.4.13
```
Comment (emord) [inline requirements/requirements.txt]: no tag exists for 0.4.10...

https://github.com/ahupp/python-magic/compare/0.4.6...0.4.13

File: requirements/requirements.txt [hunk 6]
```diff
@@ -36,19 +36,19 @@ pycrypto==2.6.1
...
 pytz==2015.6
-PyYAML==3.11
+PyYAML==3.12
```
Comment (emord) [inline requirements/requirements.txt]: http://pyyaml.org/wiki/PyYAML#History

File: requirements/requirements.txt [hunk 7]
```diff
@@ -36,19 +36,19 @@ pycrypto==2.6.1
...
 requests==2.10.0
 suds-jurko==0.6
 tropo-webapi-python==0.1.3
 xlutils==1.4.1
-Unidecode==0.04.18
+Unidecode==0.04.20
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/avian2/unidecode/blob/master/ChangeLog

File: requirements/requirements.txt [hunk 8]
```diff
@@ -36,19 +36,19 @@ pycrypto==2.6.1
...
 xlrd==0.7.3
 simplejson==2.6.1
 sh==1.09
 gevent==1.0.2
-greenlet==0.4.9
+greenlet==0.4.12
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/python-greenlet/greenlet/compare/0.4.9...0.4.12

File: requirements/requirements.txt [hunk 9]
```diff
@@ -57,7 +57,7 @@ amqp==1.4.7
 amqplib==1.0.2
 sqlagg==0.7.0
 django-redis==4.2
-redis==2.10.3
+redis==2.10.5
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/andymccurdy/redis-py/blob/master/CHANGES

File: requirements/requirements.txt [hunk 10]
```diff
@@ -66,32 +66,32 @@ stripe==1.12.2
 django-compressor==2.1
 django-angular==0.7.16
 Pycco==0.3.0
-django-mptt==0.8.6
+django-mptt==0.8.7
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/django-mptt/django-mptt/compare/0.8.6...0.8.7

File: requirements/requirements.txt [hunk 11]
```diff
@@ -66,32 +66,32 @@ stripe==1.12.2
...
 jsonpath-rw==1.4.0
 pip>=6.1.1
 git+git://github.com/smartfile/django-transfer.git@6e0dc94c3341c358fca8eb2bf74e23aee3983ec4
 Pygments==2.0.2
-tinys3==0.1.11
+tinys3==0.1.12
```
Comment (emord) [inline requirements/requirements.txt]: no change log or tags https://github.com/smore-inc/tinys3

File: requirements/requirements.txt [hunk 12]
```diff
@@ -66,32 +66,32 @@ stripe==1.12.2
...
 django-formtools==2.0
 django-two-factor-auth==1.6.0
 datadog==0.15.0
 django-websocket-redis==0.4.6
-django-redis-sessions==0.5.0
+django-redis-sessions==0.5.6
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/martinrusev/django-redis-sessions/blob/master/CHANGELOG.md change log is new. no tags

File: requirements/requirements.txt [hunk 13]
```diff
@@ -66,32 +66,32 @@ stripe==1.12.2
...
 SQLAlchemy==1.0.17
 alembic==0.8.6
 git+git://github.com/dimagi/pyzxcvbn.git#egg=pyzxcvbn
 django-statici18n==1.3.0
-django-simple-captcha==0.5.1
+django-simple-captcha==0.5.5
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/mbi/django-simple-captcha/blob/master/CHANGES

File: requirements/requirements.txt [hunk 14]
```diff
@@ -66,32 +66,32 @@ stripe==1.12.2
...
 httpagentparser==1.7.8
 boto3==1.4.4
 simpleeval==0.8.7
 laboratory==0.2.0
 ConcurrentLogHandler==0.9.1
 github3.py==1.0.0a4
 django-bulk-update==1.1.10
-csiphash==0.0.4
+csiphash==0.0.5
```
Comment (emord) [inline requirements/requirements.txt]: https://github.com/zacharyvoase/python-csiphash/compare/v0.0.4...v0.0.5

File: requirements/requirements.txt [hunk 15]
```diff
@@ -66,32 +66,32 @@ stripe==1.12.2
...
 json-delta==2.0
-email_validator==1.0.1
+email_validator==1.0.2
```
Comment (emord) [inline requirements/requirements.txt]: no changelog or tags https://github.com/JoshData/python-email-validator

**Review by emord — COMMENTED** [review]
Added links to change logs (or lack thereof) will go through and check them off with a 👍

**Comment by emord** [comment]
I think this is good to go after tests. new buddy @calellowitz

**Pull request merged**